src/main/java/org/fundacionjala/coding/wilson/Dna.java

```diff
@@ -0,0 +1,60 @@
+package org.fundacionjala.coding.wilson;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ */
+public final class Dna {
+    /**
+     *
+     */
+    private Dna() {
+
+    }
+
+    /**
+     *
+     * @param dna Given type String
+     * @return chars String
+     */
+    public static String makeComplement(final String dna) {
+        char[] arrayChar = dna.toCharArray();
+        StringBuilder dnaComplementary = new StringBuilder();
+        for (char ch : arrayChar) {
+            dnaComplementary.append(DNA.get(ch));
+        }
+        return dnaComplementary.toString();
+    }
+
+    /**
+     *
+     * @param c is char
+     * @return change
+     */
+    private static char getComplement(char c) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getComplement" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)